File: oraclecloud-sdk/src/test/java/io/micronaut/oraclecloud/mock/MockData.java
package io.micronaut.oraclecloud.mock;

import java.util.ArrayList;
import java.util.List;

public class MockData {

    public static final List<String> bucketNames = new ArrayList<>();
    public static String namespace = "test-namespace";
    public static String tenancyId = "test-tenancyId";

    public static void reset() {
        bucketNames.clear();
        namespace = "test-namespace";
        tenancyId = "test-tenancyId";
    }
}

File: docs-examples/example-function-kotlin/src/test/kotlin/example/mock/MockData.kt
package example.mock

object MockData {

    @JvmField
    val bucketNames: MutableList<String?> = ArrayList()

    @JvmField
    var namespace: String = "test-namespace"

    @JvmField
    var tenancyId: String = "test-tenancyId"
}
